interview-questions.md

```diff
@@ -16,33 +16,38 @@ Researched answer:
 
 1. What is git? What is the difference between git and Github?
 
-Your answer:
+Your answer: Git is an application that works locally through an operating system, it allows users to make code and have it work with other applications, such as text editors to do so. 
+Github is a global platform, which allows people to share commits of their code and work on it together, even work on it open source. Users make accounts and repositories and can update their own information in their own about me section/pictures to personalise it.
 
-Researched answer:
+Researched answer: Git is a free version control system that tracks changes in computer files. It is mostly used for programmers to work on source code collabortatively on non-linear devolpment. 
+Github is a version control system, however it is web based and a great collaboration platform. It can host many types of code and allow people to work on code anywhere.
 
 2. Which JavaScript operators will return a Boolean value?
 
-Your answer:
+Your answer: ===/== and conditionals? To see if True/False? 
 
-Researched answer:
+Researched answer:  Comparison operators, logical OR operators, logical AND operators, or bang opereators to even flip that. (Bonus) Or you could also use a boolean function, however that's not an operator.
 
 1. What is the difference between an array's index and its value?
 
-Your answer:
+Your answer: The index is the count of the individual data piece within the element. The value is what the pice of whole data is or will be returned.
 
-Researched answer:
+Researched answer: an arrays index is numeric, identifying the element of interest by it's ordinal position in the array (with 1 actually being 0).
+a value can be mixed types of data stored in the array. The value is found by using one of .js's built in methods.
 
 4. Compare and contrast arrays and strings. Please include ways in which they are the same and ways in which they are different.
 
-Your answer:
+Your answer: They're similar because they can both be listed and returned in the same way if coded to do so. They can both also contain just strings. However arrays can hold other data types, where as a string can only hold what is it told inside it's quotations which aren't truly different data types.
 
-Researched answer:
+Researched answer: Arrays are used to group together data of the same type, arrays are accessed by their index, which has an interger value representing it's position in the array. arrays are mutable, which means their values can be altered at any time.
```

Comment:
Arrays do not have to group together data of only the same type. For example, we can have this array with multiple data types in it: `["hello", 8, true, undefined]`
In this array, we have 4 different data types all grouped together in a single array.